api/config/v1alpha1/envoygateway_types.go

```diff
@@ -42,6 +42,12 @@ type EnvoyGatewaySpec struct {
 	// +optional
 	Provider *EnvoyGatewayProvider `json:"provider,omitempty"`
 
+	// Logging defines logging parameters for Envoy Gateway.
+	//
+	// +optional
+	// +kubebuilder:default={default: debug, gateway-api: warn}
```

Comment:
too much noise for debug, prefer to `info`

Reply:
+1

Reply:
main branch:

https://github.com/envoyproxy/gateway/blob/7f8a4913b78b492c192b85262344dce4f4f64314/internal/log/log.go#L6-L20
![image](https://github.com/envoyproxy/gateway/assets/2174082/59d45d3c-13f7-45c1-9e7a-fe0e4ece1bb8)